engines/wintermute/base/base_file_manager.cpp

```diff
@@ -372,6 +372,14 @@ uint32 BaseFileManager::getPackageVersion(const Common::String &filename) {
 
 //////////////////////////////////////////////////////////////////////////
 bool BaseFileManager::hasFile(const Common::String &filename) {
+	Common::String backwardSlashesPath = filename;
+	// correct slashes
+	for (uint32 i = 0; i < backwardSlashesPath.size(); i++) {
+		if (backwardSlashesPath[(int32)i] == '/') {
+			backwardSlashesPath.setChar('\\', (uint32)i);
```

Comment:
Unnecessary cast.

Reply:
This was copied from the existing ScummVM code in openPkgFile, meaning the loop is duplicated now. I didn't introduce a new function to leave the code as is as much as possible. Cast is only removed in hasFile now. Still make an extra function out of it?

Reply:
Nevermind, algorithm.h has replace